lxc/query.go

```diff
@@ -109,7 +109,9 @@ func (c *cmdQuery) Run(cmd *cobra.Command, args []string) error {
 
 		// If not JSON decoding error then fail immediately.
 		if !errors.As(err, &jsonSyntaxError) && !errors.As(err, &jsonUnmarshalTypeError) {
-			return err
+			if (err.Error() != "EOF") {
```

Comment:
This is relevant for this PR, please can you remove and if its related to a different issue perhaps open an issue on github to discuss?